app/display.py

```diff
@@ -0,0 +1,31 @@
+from abc import ABC, abstractmethod
+
+from typing import TYPE_CHECKING
+if TYPE_CHECKING:
+    from app.main import Book
+
+
+class BookDisplay(ABC):
+    @abstractmethod
+    def __init__(self, book: "Book") -> None:
+        self.book = book
+
+    @abstractmethod
+    def display(self) -> None:
+        pass
+
+
+class ConsoleBookDisplay(BookDisplay):
+    def __init__(self, book: "Book") -> None:
```

Comment:
Do not use quotes in the type annotation, remove them
```suggestion
    def __init__(self, book: Book) -> None:
```
In case you face errors, use `from __future__ import annotations`. Apply this change in all places

Reply:
Thanks for advice. I had a problem with circular imports but it fixed this.